gcloud/_apitools/http_wrapper.py

```diff
@@ -91,8 +90,7 @@ def _Httplib2Debuglevel(http_request, level, http=None):
     httplib2.debuglevel = old_level
     if http is not None:
         for connection_key, old_level in http_levels.items():
-            if connection_key in http.connections:
-                http.connections[connection_key].set_debuglevel(old_level)
+            http.connections[connection_key].set_debuglevel(old_level)
```

Comment:
It is certainly possible but I'm not familiar enough with the rest of the library to say what is safe here.

In most cases, you'd say "if the user removes the key before re-entry, they deserve the `KeyError`" but I don't know what sorts of operations users could perform or (more importantly) what those operations hide from the users.